### PR TITLE
JBPM-6891 - Custom object in QueryParamBuilder

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/rest/RestURI.java [hunk 1]
```diff
@@ -209,6 +209,7 @@
     public static final String QUERY_DEF_GET_URI = "{" + QUERY_NAME + "}";
     public static final String RUN_QUERY_DEF_GET_URI = "{" + QUERY_NAME + "}/data";
     public static final String RUN_FILTERED_QUERY_DEF_POST_URI = "{" + QUERY_NAME + "}/filtered-data";
+    public static final String RUN_FILTERED_QUERY_DEF_BY_CONTAINER_POST_URI = "containers/{" + CONTAINER_ID + "}/queryname/{" + QUERY_NAME + "}/filtered-data";
```
Comment (mswiderski) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/rest/RestURI.java]: just one comment - please replace queryname with query

**Comment by tsurdilo** [comment]
@mswiderski updated and fixed conflict. thanks.

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**